ls1intum/Artemis
# [Feature] Show dates on exam cover, when the exam spans multiple days

### Checklist

### Motivation and Context


Fix #3691 

### Description

Shows the date of start and end time of an exam on the cover, when they are different.

### Steps for Testing


1. Log in to Artemis
2. Create two exams. One that starts and ends on the same day, another that spans multiple days
3. Go to the start page of these exams as a student.
4. Check that the dates and times are displayed correctly and are intuitively

### Screenshots



An exam that only lasts for one day:
![Bildschirmfoto 2021-07-07 um 22 38 10](https://user-images.githubusercontent.com/38322605/124825974-94c3ea80-df74-11eb-97ec-cfd900747afa.png)

An exam that lasts multiple days:
![Bildschirmfoto 2021-07-07 um 22 35 33](https://user-images.githubusercontent.com/38322605/124825975-955c8100-df74-11eb-9a15-175d03ada0e2.png)
